### DISCUSSION TITLE
alarm sound copyrights

### DISCUSSION THREAD

**Discussion by khaledmurtaja** [gdelataillade/alarm#264] [Q&A]
Where to get sounds for the alarm which will not violate playstore/app store rules? like where i can get some good sounds
is the sounds on the example could be safely used?

**Comment by gdelataillade** [comment]
No it's not safe, most of the sounds in the example are probably protected. You should try [Soundcloud](https://soundcloud.com). Or just Google "free mp3 sounds".
